src/Styles.js

```diff
@@ -66,11 +66,17 @@ export default css`
 }
 
 anypoint-button[active] {
+  color: var(--api-body-document-media-button-color);
```

Comment:
Was it already using these values? Or are we changing the defaults?
I don't see defaults specified in the var, so wasn't sure if we're going to impact other consumers.

Reply:
In this case, I'm defining a new var (`api-body-document-media-button-color`), so existing consumers wouldn't have a value defined for this. So by default, it should be undefined and have no effect